README.md

```diff
@@ -6,13 +6,21 @@ The recipes folder contains all the recipes in this repo in no particular order.
 
 ## Contributing new recipes
 
+### Principles
+
+#### Only use packages in the latest package set release
+
+In other words, we do not allow anyone to override or add packages in the `packages.dhall` file. If one overrided a given package to make Recipe A work, it might break Recipe B that also relies upon that package. If one added a package that isn't later maintained, it might prevent our CI from passing, which can delay new recipes from being added to this cookbook.
```

Comment:
Thinking this might read better as bullets:
> * Overriding a package to make Recipe A work might break Recipe B ...
> * Packages outside of the package set are more likely to be neglected ...

The first situation would be caught by CI though.

A more realistic scenario is Recipe A making it through CI with an overridden package, which then _prevents_ a new Recipe B from passing CI with the default package set. Maybe that's the situation you have in mind.

Also wondering about overrides to more recent versions of packages already tracked in the package set. For example can new Halogen Hooks recipes get merged while we're waiting for the latest version to be picked up? I guess it's fair to just those recipe PRs sit until the corresponding package set PR is merged.

Reply:
> Also wondering about overrides to more recent versions of packages already tracked in the package set. For example can new Halogen Hooks recipes get merged while we're waiting for the latest version to be picked up? I guess it's fair to just those recipe PRs sit until the corresponding package set PR is merged.

Yeah, we'd wait to merge it until later. 

> Thinking this might read better as bullets:

I agree.

> The first situation would be caught by CI though.
> 
> A more realistic scenario is Recipe A making it through CI with an overridden package, which then prevents a new Recipe B from passing CI with the default package set. Maybe that's the situation you have in mind.

That's true. To clarify more points, I think there's a maintenance burden we don't pay for if we only support recipes which can run using the regular package set. The other concern is if someone adds/overrides a malicious package. While we don't have a strong vetting process, it at least goes through more eyes this way.

Reply:
I've accounted for your feedback. Please let me know your thoughts.